src/main/scala/com/linagora/openpaas/gatling/unifiedinbox/scenari/InboxScenari.scala

```diff
@@ -28,8 +27,7 @@ object InboxScenari {
         group("INBOX")(
           exec(InboxScenari.userLogin())
             .exec(repeat(20) {
-              exec(PKCESteps.renewAccessToken)
-                .randomSwitch(
+                randomSwitch(
```

Comment:
Shouldn't the Inbox simulation also have a step to renew token just like Calendar?

```suggestion
              exec(LoginSteps.renewTokenIfNeeded)
                .randomSwitch(
```